pyproject.toml

```diff
@@ -55,3 +56,11 @@ CI-CD = [
   "cubitpy@git+https://github.com/imcs-compsim/cubitpy.git@main",
   "setuptools" # Needed for coverage-badge
 ]
+
+[tool.pytest.ini_options]
+addopts = "-p pytest_cov --cov-report=term --cov-report=html --cov-fail-under=30 --cov=meshpy/ --cov=tutorial/ --cov-append"
```

Comment:
Why only 30%? I think we can set that to 90. (With the exception of the performance tests).

Should we also add the `tests/` directory here? I personally think it makes sense to identify untested code in the testing framework.

Reply:
Locally I only execute the pytest framework and there I have 38% coverage - that's why I set it that low.

Normally the `tests/` is not added as only the coverage of the source code is of intereset